#Set the parameters of the robot system
wheel_radius: 0.062
robot_wheelbase: 0.330

#Set parameters of the simulation node
sim_sample_time: 0.02
sim_node_rate: 100

#Set parameters of the localisation node
localisation_sample_time: 0.02
localisation_node_rate: 100

#Set parameters of the Joint Publisher node
joint_sample_time: 0.02
joint_node_rate: 100

sim1:
  #Set initial conditions of the robot 1
  pos_x0: 0.00
  pos_y0: 0.00
  pos_th0: 0.00

sim2:
  #Set initial conditions of the robot 2
  pos_x0: 0.00
  pos_y0: 0.00
  pos_th0: 0.00